### ISSUE TITLE
[conditions/remove-zero]: wrong autofix on conditions with `%`

### ISSUE THREAD

**Issue by ElPrudi** [coderaiser/putout#157]
I love the idea of this rule to shorten my code, but sadly it doesn't correctly work with `%`. For example, it changes

```ts
if ((a - b) % 2 === 0)
``` 

incorrectly to:

```ts
if (!(a - b) % 2)
```

which leads to this TypeScript error:
```
The left-hand side of an arithmetic operation must be of type 'any', 'number', 'bigint' or an enum type.
```

and evaluates to:

```ts
if (false % 2)
```

It should actually fix to:

```ts
if (!((a - b) % 2))
```

**Comment by coderaiser** [comment]
Thanks, fixed with 4fa8f58, landed in `@putout/plugin-conditions@2.2.0` 🎉. Please re-install 🐊**Putout**, is it works for you?

**Comment by ElPrudi** [comment]
Can confirm, it has been fixed. Thank you very much :)
